### PR TITLE
fix(registry): Add certificate to OpenShift objects to be able to pull after ICSP

### LINKED ISSUE #382: ManagedCluster doesn't report as ManagedClusterConditionAvailable during detach
image

### LINKED ISSUE #383: Review registry trusting process
Currently, the trust_internal_registry just adds the CA to the host itself, but documentation:

https://docs.openshift.com/container-platform/4.10/cicd/builds/setting-up-trusted-ca.html

States that a configmap should be created and used to patch the OpenShift image config

Related with #382

### PR DESCRIPTION
**Pull request by iranzo**
# Description

The certificate of  registry is added to the host trust in the prior steps but never gets added to OpenShift itself

Fixes #382 
Fixes #383

### CODE REVIEW

File: deploy-disconnected-registry/common.sh [hunk 1]
```diff
@@ -90,6 +90,11 @@ function trust_internal_registry() {
     update-ca-trust extract
     echo ">> Done!"
     echo
+
+    # Add certificate to OpenShift configuration
+    oc --kubeconfig=${KBKNFG} create configmap ztpfwregistry -n openshift-config --from-file=ztpfw-registry-ztpfw-registry.apps.test-ci.alklabs.local:5000=${PATH_CA_CERT}
```
Comment (achuzhoy) [inline deploy-disconnected-registry/common.sh:95]: Will this particular FQDN actually fix the issue we see running the pipeline?

Comment (iranzo) [reply deploy-disconnected-registry/common.sh]: No, it was set to ztpfw to add the nickname, it's updated in the current PR

**Comment by alknopfler** [comment]
Juan did it and the result of the investigation was create the current workflow. With the first approach (mco) it takes over 1h more than now

**Review by fbac — APPROVED** [review]
LGTM.

Let pass some test so we can check the time this change takes.

**Review by fbac — APPROVED** [review]
lgtm

**Pull request merged**